src/test/java/apicurito/tests/configuration/TestConfiguration.java

```diff
@@ -52,6 +53,7 @@
     public static final String QUAY_NAMESPACE = "apicurito.config.quay.namespace";
     public static final String QUAY_TOKEN = "apicurito.config.quay.auth.token";
     public static final String QUAY_OPSRC_TOKEN = "apicurito.config.quay.opsrc.token";
+    public static final String QUAY_PULL_SECRET = "apicurito.config.quay.pull.secret";
```

Comment:
is this some other pull secret than https://github.com/Apicurio/apicurio-qe/pull/61/files#diff-b81094bfde713d8bd4bb083970afdbff5a188b45e4cd3d20c95a89c5731570d4R46 ?

Reply:
the link might not be working that well, I mean the pull secret on line #46 (APICURITO_PULL_SECRET)?

Reply:
The value itself is definitely different. It is used for pulling bundle resources - which required some openshift images and also our custom bundle images (if we ever decide to make them private). This pull secret can be obtained from [https://cloud.redhat.com/openshift/install/crc/installer-provisioned](https://cloud.redhat.com/openshift/install/crc/installer-provisioned)

Reply:
yes as Martin said there are two different pull secrets for the quay.

Reply:
ok, thanks for the clarification